docker_entrypoint.sh

```diff
@@ -1,4 +1,14 @@
 #!/bin/sh
+
+### 1. Adds local user (UID and GID are provided from environment variables).
+### 2. Updates permissions, except for ./public/system (should be chown on previous installations).
```

Comment:
It would take way too long even with my optimised command. I can try with GNU parallel but this seems a bit hacky IMO... 🤔